draft-ietf-quic-recovery.md

```diff
@@ -836,9 +836,11 @@ in Linux (3.11 onwards).
 
 ## Under-utilizing the Congestion Window
 
-A congestion window that is under-utilized SHOULD NOT be increased in either
-slow start or congestion avoidance. This can happen due to insufficient
-application data or flow control credit.
+When bytes in flight is smaller than the congestion window and sending is not
+pacing limited, the congestion window is under-utilized.  When this occurs,
```

Comment:
Where does the pacing limitation come from?

Reply:
There are congestion controllers now, and hopefully more in the future, that operate primarily pacing limited, not CWND limited, so this was an effort to capture that, but it's certainly imperfect.  Any suggestions?

Reply:
I think BBR is considered here for pacing limited case? But there is no description what is pacing limited.

Reply:
@junhochoi : The point here is that if you're unable to utilize the full cwnd because you are limited by a pacer, then it doesn't count as app limited.